python/svix/api.py

```diff
@@ -68,6 +68,11 @@ def __init__(self, configuration: Configuration) -> None:
     @contextmanager
     def _api(self) -> t.Generator[ApiClass, None, None]:
         with ApiClient(self._configuration) as api_client:  # type: ignore
+
+            import svix
```

Comment:
```suggestion
            from . import __version__
```
Maybe? (make sure it works)